packages/graphql/src/testing/utils.ts

```diff
@@ -0,0 +1,12 @@
+import { GraphQLSchema } from "graphql";
+
+export const EMPTY_SCHEMA = new GraphQLSchema({});
+
+export const EMPTY_SCHEMA_OUTPUT = `#graphql
```

Comment:
It would be nice to generate this output using printSchema and GraphQL AST, rather than hardcoding as a string like we do in the prototype tests. E.g.

```
schemaWithDummyQuery = new GraphQLSchema({...})


getEmittedOutput(schemaWithDummyQuery, options):
    return printSchema(schemaWithDummyQuery)
```

This way we'll get all the spacing etc correct.

Reply:
Agreed. I have done that [in later work](https://github.com/pinterest/typespec/pull/2) but forgot to reconcile it here.